tests/util.py

```diff
@@ -47,15 +47,27 @@ async def protoc(
     return stdout, stderr, proc.returncode
 
 
-def get_test_case_json_data(test_case_name: str, json_file_name: Optional[str] = None):
-    test_data_file_name = json_file_name or f"{test_case_name}.json"
-    test_data_file_path = inputs_path.joinpath(test_case_name, test_data_file_name)
+def get_test_case_json_data(test_case_name: str, *json_file_names: str) -> List[str]:
```

Comment:
this change makes it easier to manage multiple sample message json files in the test inputs.